girder/api/v1/file.py

```diff
@@ -266,7 +268,15 @@ def download(self, file, params):
             if endByte is not None:
                 endByte = int(endByte)
 
-        return self.model('file').download(file, offset, endByte=endByte)
+        contentDisp = params.get('contentDisposition', None)
+        allowed = ['inline', 'attachment']
+        if contentDisp is not None and contentDisp not in allowed:
+            raise RestException('Unallowed contentDisposition type "%s".' %
+                                contentDisp)
+
+        return \
+            self.model('file').download(file, offset, endByte=endByte,
```

Comment:
To avoid backslash here you can break after the opening paren `download(` and just indent the next line.